instaloader/instaloader.py

```diff
@@ -204,6 +206,26 @@ def __init__(self,
         self.resume_prefix = resume_prefix
         self.check_resume_bbd = check_resume_bbd
 
+        self.slide = slide or ""
+        self.slide_start = 0
+        self.slide_end = 0
+        if self.slide != "":
+            splitted = self.slide.split('-')
+            try:
+                if len(splitted) == 1:
+                    self.slide_start = self.slide_end = int(splitted[0])
+                    if self.slide_start < 0:
```

Comment:
```suggestion
                    if self.slide_start <= 0:
```

slides start counting at 1.